filebeat/harvester/log.go

```diff
@@ -297,6 +313,36 @@ func (h *Harvester) handleReadlineError(lastTimeRead time.Time, err error) error
 func (h *Harvester) Stop() {
 }
 
+func InitRegexp(exprs []string) []*regexp.Regexp {
+
+	result := []*regexp.Regexp{}
+
+	for _, exp := range exprs {
+
+		rexp, err := regexp.Compile(exp)
+		if err != nil {
+			logp.Err("Fail to compile the regexp %s: %s", exp, err)
+			continue
```

Comment:
maybe we should return an error. We really want to run with incomplete filter setup?